Sources/ServerSideIncludes.php

```diff
@@ -1039,7 +1066,7 @@ public static function topTopics($type = 'replies', $num_topics = 10, $output_me
 	 * @param string $output_method The output method. If 'echo', displays a list of topics, otherwise returns an array of info about them
 	 * @return void|array Either displays a list of top topics or returns an array of info about them, depending on output_method.
 	 */
-	public static function topTopicsReplies($num_topics = 10, $output_method = 'echo')
+	public static function topTopicsReplies(int $num_topics = 10, string $output_method = 'echo'): array
```

Comment:
This must be returned typed for ?array since it returns self::topTopics, which could return null